### DISCUSSION TITLE
Line 86 may not correct in README.md

### DISCUSSION THREAD

**Discussion by sxming8737** [z4kn4fein/php-semver#5] [Show and tell]
Hi Peter,

In file [README.md](https://github.com/z4kn4fein/php-semver/blob/master/README.md), I found the line 86 may not correct.
Could you please help check it?
echo Version::compareString("2.3.4", "3.1.0");                            // 1, it actually is -1 in my test.
Thanks

**Comment by z4kn4fein** [comment]
> Hi Peter,
> 
> In file [README.md](https://github.com/z4kn4fein/php-semver/blob/master/README.md), I found the line 86 may not correct. Could you please help check it? echo Version::compareString("2.3.4", "3.1.0"); // 1, it actually is -1 in my test. Thanks

Hi @sxming8737, you're right, I've fixed it. Thanks for the report!
